### DISCUSSION TITLE
My computer turns on as soon as it turns off

### DISCUSSION THREAD

**Discussion by dantin-durand** [AlexGustafsson/homebridge-wol#248] [Questions]
Hello everyone !
I have a problem with the plugin I don't know if it's because of a bad configuration of my computer or a bug with the plugin but recently I noticed that as soon as I turn off my PC it turns back on directly and when I deactivate the plugin I no longer have the problem..

here is my config on the plugin:
```
{
    "name": "PC name",
    "manufacturer": "homebridge-wol",
    "model": "NetworkDevice",
    "host": "192.168.1.50",
    "pingInterval": 2,
    "pingsToChange": 5,
    "pingTimeout": 1,
    "pingCommandTimeout": 0,
    "mac": "my MAC adress",
    "startCommandTimeout": 0,
    "wakeGraceTime": 45,
    "wakeCommandTimeout": 0,
    "shutdownCommand": "net rpc shutdown -I 192.168.1.50 -U ssh%mypassword -f",
    "shutdownGraceTime": 30,
    "shutdownCommandTimeout": 0,
    "logLevel": "Info",
    "returnEarly": false,
    "accessory": "NetworkDevice"
}
```

Thank you in advance for your help !

**Comment by dantin-durand** [comment]
up

**Comment by Zlango** [comment]
Since a few days I have the same error

**Comment by AlexGustafsson** [comment]
Did you end up finding a solution for this?

**Reply by dantin-durand** [reply]
Nop I had to deactivate the plugin 😕

**Reply by AlexGustafsson** [reply]
Sorry to hear that. It sounds a bit like your computer had some "wake up on network access" feature. As if it wakes up whenever it is pinged.
